ddtrace/profiling/collector/_memalloc.c

```diff
@@ -35,29 +46,27 @@ static void
 memalloc_add_event(memalloc_context_t* ctx, void* ptr, size_t size)
 {
     /* Do not overflow; just ignore the new events if we ever reach that point */
-    if (global_traceback_list->alloc_count >= UINT64_MAX)
+    if (global_alloc_tracker->alloc_count >= ALLOC_TRACKER_MAX_COUNT)
```

Comment:
```suggestion
    if (global_alloc_tracker->alloc_count == ALLOC_TRACKER_MAX_COUNT)
```

I kinda get why `>=` is used but there shouldn't be a case where `global_alloc_tracker->alloc_count > ALLOC_TRACKER_MAX_COUNT` (especially if it's `UINT64_MAX`)

I don't feel strongly about this though

Reply:
Yep, it's just a convention write >=, but it can't be > as you said.